setup.py

```diff
@@ -33,7 +33,7 @@
         "backoff",
         "click",
         "dacite",
-        "fastapi-azure-auth",
+        "fastapi-azure-auth==4.0.0",
```

Comment:
Why was it required to lock this version?

Reply:
New version of this package had breaking changes, causing isar to crash when launching